cypress/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "lib": ["es2015", "es2017", "dom"],
+    "strict": true,
```

Comment:
Heh. Ooops. We never really had type checking on Cypress files... I had to add libs to get Typescript to pass on Cypress types since they use types from the dom lib.